city/routers.py

```diff
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, Depends, Response, status
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from settings import settings
+from city import schemas, crud
+from dependencies import get_db
+
+
+prefix = settings.API_PREFIX
+router = APIRouter()
+
+
+@router.get(f"{prefix}/cities/", response_model=list[schemas.City])
+async def read_cities(db: AsyncSession = Depends(get_db)):
```

Comment:
The endpoint `/temperatures/update/` seems to be related to temperature data. If this file is specifically for city routers, consider moving this endpoint to the appropriate module handling temperature data.